what-is-the-ic/3_canisters/index.md

```diff
@@ -4,16 +4,11 @@ card: /img/what-is-the-ic/canister.jpg
 cardImageFit: cover
 ---
 
-A smart contract is a computer program executed on a blockchain. A *canister*, or canister smart contract, is a bundle comprising a computer program and its data. Every canister is hosted on one subnet of the IC.
+A smart contract is a computer program executed on a blockchain. A canister is a smart contract that bundles a computer program and its data. Every canister is hosted on one subnet of the IC.
+Canisters can be executed concurrently and communicate within and across subnets by sending asynchronous messages in a non-blocking manner, resulting in essentially unbounded scalability.
+Canisters are more powerful than smart contracts on other blockchains as they can
+* serve a user interface directly to any web browser,
+* hold gigabytes of memory for a low fee, perform substantial amounts of computation at a low cost, and pay for their own computation (learn more about the [reverse gas model](https://internetcomputer.org/features/reverse-gas/)).
```

Comment:
```suggestion
* hold gigabytes of memory for a low fee
* perform substantial amounts of computation at a low cost, and
* pay for their own computation (learn more about the [reverse gas model](https://internetcomputer.org/features/reverse-gas/)).
```

Reply:
I believe we were trying to reduce the number of bullets, but I agree it's tidier. @THLO @dietersommer  do you have an opinion?

Reply:
Had the same comment yesterday why there are only 2 bullets. ;-)

Reply:
I also agree that it looks better with the bullets. I'm in favor of accepting the proposed change.